container-storage-setup.sh

```diff
@@ -78,6 +78,9 @@ _STORAGE_OPTIONS="STORAGE_OPTIONS"
 # Keeps track of if we created a volume group or not.
 _VG_CREATED=
 
+# Do we have parted
+_PARTED=$(which parted || echo "")
```

Comment:
There is no need to create a global variable for this information. Its not re-used outside a single function. How about creating a helper function say `has_parted` and use that to decide whether to use parted for sfdisk.

if has_parted; then
   use_parted
else
  use_sfdisk